### ISSUE TITLE
Albumentation 1.1.0

### ISSUE THREAD

**Issue by KC-decoder** [ubccr/software-layer#320]
Hello,
I would like to request CCR to have the Python library Albumentations AI be build through Easybuild. The recipe can be found here:

[https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/a/Albumentations/Albumentations-1.1.0-foss-2021b.eb]

**Comment by tonykew** [comment]
Albumentation has been built, and is going through internal testing.
I will update this request once Albumentation has been published.

**Comment by tonykew** [comment]
Albumentation version 1.1.0 has been published to the ccrsoft/2023.01 software release

```
login2$ module spider Albumentation
----------------------------------------------------------------------------
  albumentations: albumentations/1.1.0
----------------------------------------------------------------------------
    Description:
      Albumentations is a Python library for fast and flexible image
      augmentations


    You will need to load all module(s) on any one of the lines below before the "albumentations/1.1.0" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
login2$ module load gcc/11.2.0 openmpi/4.1.1 albumentations/1.1.0
login2$ ls ${EBROOTALBUMENTATIONS}/lib/python3.9/site-packages
albumentations  albumentations-1.1.0.dist-info  qudida  qudida-0.0.4.dist-info
login2$ 
```

Loading the module adds albumentations to the PYTHONPATH

```
login2$ echo ${PYTHONPATH} | grep ${EBROOTALBUMENTATIONS}/lib/python3.9/site-packages
/cvmfs/soft.ccr.buffalo.edu/versions/2023.01/easybuild/software/avx512/MPI/gcc/11.2.0/openmpi/4.1.1/albumentations/1.1.0/lib/python3.9/site-packages:/cvmfs/s[...]
login2$ 
```
